docs/migration.md

```diff
@@ -281,3 +281,95 @@ metadata:
     name: fio-cstor
     openebs.io/target-affinity: fio-cstor
 ```
+
+# Migrating jiva External Provisioned volumes to jiva CSI volumes (Experimental)
+
+These instructions will guide you through the process of migrating jiva volumes from the old v1apha1 external provisioned spec to v1 CSI spec. 
+
+### Prerequisites
+
+Before migrating the volumes make sure the following prerequisites are taken care of:
+
+ - The jiva-operator should be installed with version 2.7.0 or above. You can install the correct version of jiva-operator from [charts](https://github.com/openebs/charts/tree/gh-pages). Get the jiva-operator yaml within the correct versioned folder and install. 
```

Comment:
Could you instead provide the link to the latest version of the jiva-operator.yaml?

Reply:
Users may not want to go the latest version and want a specific version. This page will have a jiva-operator.yaml at the root folder which will always point to the latest yaml.